lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -188,11 +191,12 @@ def add_tunnel_flows(self, precedence:int, i_teid:int,
         # Install Downlink Tunnel
         actions = []
         if ue_ip_adr:
+            # Add flow for LOCAL port
```

Comment:
Will this ever not have a ue_ip_adr? 
If so can we just do
```
if not ue_ip_adr:
    self.logger.error(...)
    return
```

Reply:
Yes.. It's correct way of implementation for error checking. Changes done.